canvas_modules/common-canvas/src/common-properties/controls/checkboxset/checkboxset.jsx

```diff
@@ -97,12 +98,21 @@ class CheckboxsetControl extends React.Component {
 			/>);
 		}
 		return (
-			<div className={classNames("properties-checkboxset", { "hide": this.props.state === STATES.HIDDEN })} data-id={ControlUtils.getDataId(this.props.propertyId)} >
-				<div className="properties-checkboxset-container">
-					{checkboxes}
+			<fieldset className="bx--fieldset">
```

Comment:
I don't really like that we are adding fieldset and legend here.  Will we have to do this for each control?

Reply:
This is how checkboxset heading is specified in [carbon documentation](https://www.carbondesignsystem.com/components/checkbox/code/#live-demo). 
We have to do something similar for [radio button group](https://www.carbondesignsystem.com/components/radio-button/code/#live-demo) as well. (Radioset control in properties). 
![Screen Shot 2020-12-03 at 10 57 00 AM](https://user-images.githubusercontent.com/25124000/101074999-4ec62480-3556-11eb-9b04-9c770ba78a6f.png)

Reply:
How about numbers, textfield, etc...?

Reply:
Text input, Dropdown, Number input, TextArea - All these components have `labelText` or `label` attributes to specify name of the control.